git_downloader.py

```diff
@@ -30,18 +38,18 @@ def main(args, loglevel):
                     if os.path.exists(output_path):
                         output_path += "-" + str(file_counter)
                     try:
-                        urllib.urlretrieve(file, output_path)
-                    except:
-                        logging.error('Error downloading %s' % file)
-            except:
-                 logging.error('Error fetching repository %s' % line)
+                        urlretrieve(file, output_path)
+                    except Exception as error:
+                        logging.error('Error downloading %s. %s' % (file, error))
```

Comment:
`logging.exception` just log the message in error level and prints the enclosing exception info.
It's equivalent to `logging.error('My message', exc_info=True) `